content/pages/roadmap.html

```diff
@@ -16,25 +16,187 @@ <h4><i class="fa fa-map"></i> <b>Roadmap</b></h2>
 
                 <p>If you have an opinion about where iRODS should go next, I encourage you to participate in this open software development process. Join the <a href="https://groups.google.com/forum/#!forum/irod-chat">iRODS chat Google Group</a>. Raise issues on <a href="https://github.com/irods/irods">GitHub</a>. And if you are a <a href="{filename}/pages/consortium-membership.html">Consortium Member</a>, get involved in the Technology Working Group.</p>
 
-                <p>Terrell Russell<br/>
+                <p>Kory Draughn<br/>
                 Chief Technologist<br/>
-                The iRODS Consortium<br/>
-                <i>(Last Updated March 2017)</i></p>
+                The iRODS Consortium</p>
 
                 <br/><br/>
-                <hr/>
 
-                <h1><i class="green_font fa fa-angle-right"></i> Table of Contents</h1>
+                <h1><i class="green_font fa fa-angle-right"></i> Document History</h1>
                 <ul>
-                    <li id="upcoming"><a href="#upcoming">Upcoming Release: iRODS 4.3</a></li>
-                    <li><a href="#future">Future Direction</a></li>
-                    <li><a href="#previous">Previous Releases</a></li>
+                    <li><a href="#upcoming">2023</a></li>
+                    <li><a href="#2017">2017</a></li>
+                    <li><a href="#2015">2015</a></li>
                 </ul>
 
-                <hr/>
+                <script>
+                    function togglevisibility(divname) {
+                        var x = document.getElementById(divname);
+                        if (x.style.display === "block") {
+                            x.style.display = "none";
+                        } else {
+                            x.style.display = "block";
+                        }
+                    }
+                </script>
+
+                <br/>
+                <hr style="border-top:5px solid;color:#06cd99"/>
+                <br/>
+
+                <div id="upcoming"></div>
+                <div id="2023"></div>
+
+                <h1><i class="green_font fa fa-angle-right"></i> <a onclick="togglevisibility('2023body')">2023</a></h1>
+
+                <div id="2023body" style="display:block">
+
+                    <p>Kory Draughn<br/>
+                    Chief Technologist<br/>
+                    The iRODS Consortium<br/>
+                    <i>January 2023</i></p>
+
+                    <h2>iRODS 2023 Roadmap</h2>
+
+                    <p>With each release of iRODS, the server, libraries, and clients become more stable and featureful. The needs of users and administrators become more complex over time. Users want tools and GUIs that are easy to understand quickly. Administrators want more flexibility and visibility into what the system is doing.</p>
+
+                    <p>In order to facilitate these needs, the Consortium has three goals: Simplification, Maintainability, and Approachability.</p>
```

Comment:
How about ...

In order to facilitate these needs, the Consortium [is focused / will focus] on improving approachability.

- OR -

In order to facilitate these needs, the Consortium plans to improve approachability by focusing on the following:
_(this replaces L65)_

Thoughts?

Reply:
yes, the second is nice.

Reply:
Done.